content/physical_properties/dielectric_permittivity/dielectric_permittivity_units.rst

```diff
@@ -16,7 +16,7 @@ The units for dielectric proporties and associated quantities are summarized in
 +---------------------------+-----------------------------------+------------------+
 | Dielectric Permittivity   | :math:`\varepsilon`               | F/m              |
 +---------------------------+-----------------------------------+------------------+
-| Dielectric Susceptibility | :math:`\chi_e`                    | *Unitsless*      |
+| Dielectric Susceptibility | :math:`\chi_e`                    | *Unitless*       |
```

Comment:
I don't think Dielectric Susceptibility has been mentioned yet, so it might be worth including its mathematical relationship to dielectric permittivity